code/modules/mob/living/carbon/human/species/golem.dm

```diff
@@ -15,7 +15,7 @@
 	punchdamagelow = 5
 	punchdamagehigh = 14
 	punchstunthreshold = 11 //about 40% chance to stun
-	no_equip = list(SLOT_HUD_WEAR_MASK, SLOT_HUD_OUTER_SUIT, SLOT_HUD_GLOVES, SLOT_HUD_SHOES, SLOT_HUD_JUMPSUIT, SLOT_HUD_SUIT_STORE)
+	no_equip = list(ITEM_SLOT_MASK, ITEM_SLOT_OCLOTHING, ITEM_SLOT_GLOVES, ITEM_SLOT_SHOES, ITEM_SLOT_ICLOTHING, ITEM_SLOT_SUIT_STORE)
```

Comment:
this should be turned into a bitflag instead of a list of bits. Youll need to do some more changes with the species backend to fix it.